stac_fastapi/pgstac/setup.py

```diff
@@ -13,20 +13,22 @@
     "stac-fastapi.types",
     "stac-fastapi.api",
     "stac-fastapi.extensions",
+    "psycopg[binary]",
     "asyncpg",
     "buildpg",
     "brotli_asgi",
     "pygeofilter @ git+https://github.com/geopython/pygeofilter@v0.1.1#egg=pygeofilter",
+    "pypgstac==0.6.0",
 ]
 
 extra_reqs = {
     "dev": [
+        "pypgstac[psycopg]==0.6.0",
```

Comment:
Seems like we could move this to `install_requires` and remove `psycopg[binary]`, given that `pypgstac[psycopg]` contains pyscopyg[binary] and [pool]

Reply:
Actually, now I'm confused - aren't we avoiding psycopy in the standard install, and only including it in the dev extras? Why is `psycopg[binary]` required in `install_requires` above?

Reply:
Ah, good catch. `psychopg[binary]` was added back in https://github.com/stac-utils/stac-fastapi/pull/397/commits/efc6e6404bd8920c5dc389f0069b5ee13e167e11#diff-55ad2c09aab406c3bab2128a485d2faaabe4db8c50d3b3c8efc69a9cc8b3c4a1 and was only meant to be temporary until pypgstac was released. I'll remove it.

Reply:
Removed in 9a70a02